docs/architecture/README.md

```diff
@@ -89,7 +88,7 @@ therefore they have links to it and refer to CometBFT as "Tendermint" or "Tender
 
 ### Deprecated
 
-None
+- [ADR-035: Documentation](tendermint-core/adr-035-documentation.md)
```

Comment:
Moving this ADR into the `tendermint-core` directory is probably causing the TOC check workflow to break. I'd recommend doing the move in a separate follow-up PR, and rather keep this ADR along with all the others.

Reply:
it broke I believe because of a file permission which I fixed and the workflow executes, but I can revert this change. Makes sense to leave to an ADR refactor issue